app/assets/javascripts/posts/search.js

```diff
@@ -37,6 +37,24 @@ $(document).ready(function() {
     allowClear: true,
   });
 
+  createSelect2('#user_id', {
```

Comment:
This file is used in all our searches, right?

Would be nice to rename it to just `javascripts/search.js` at some point!

Reply:
Yeah, I can make another small PR for that later